.eslintrc.js

```
@@ -1,61 +1,304 @@
+		'no-unused-expressions': ['error', { allowShortCircuit: true, allowTernary: true }],
+		'no-useless-concat': 'error',
+		'no-useless-return': 'error',
+		radix: 'error',
+		yoda: 'error',
+		// Code style
+		'array-bracket-spacing': ['error', 'never'],
+		'block-spacing': ['error', 'always'],
+		'brace-style': ['error', '1tbs', { allowSingleLine: true }],
+		// Snake_case enforcement for Frappe/ERPNext framework compatibility
+		// Replaces camelcase rule to align with framework conventions and E-Boekhouden API
+		camelcase: ['error', { 
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)